core/handlers.py

```diff
@@ -107,6 +107,44 @@
     )
 
 
+@dp.message_handler(commands=["peek"], state="*")
+async def peek_command_handler(msg: types.Message):
+    for job in scheduler.get_jobs():
+        extra = 1 if job.id.endswith("day_before") else 0
+        job_chat_id = job.args[0]
+        if job_chat_id == msg.from_user.id:
+            remainder_is_scheduled_text = _("remainder_is_scheduled")
```

Comment:
you may add a helper function like `get_user_reminders(user_id: int) -> "apscheduler.job.Job"`, sort it by `next_run_time` and then get the first. It would be easier to read this function asaik